src/app/filter.pipe.ts

```diff
@@ -12,9 +12,15 @@ export class FilterPipe implements PipeTransform {
     }
 
     searchText = searchText.toLowerCase();
+    // split the search string into an array of individual words
+    const searchTextAsWords = searchText.match(/\s+/g) || [];
 
     return items.filter( item => {
-      return item.displayText.toLowerCase().includes(searchText);
+      // results should contain all search words
+      return searchTextAsWords.every(matchAll);
+      function matchAll(wordItem) {
```

Comment:
It might be more readable to have this be an inline, anonymous function

Reply:
I'm sure that's true.  I don't know how to do that.